modules/editorial-metadata/editorial-metadata.php

```diff
@@ -327,1114 +328,1203 @@ function register_taxonomy() {
+				die( esc_html( $change_error->get_error_message() ) );
+			}
 
-	/**
-	 * If we're ordering by a sortable column, let's modify the query
-	 *
-	 * @since 0.7.4
-	 */
-	function action_parse_query( $query ) {
+			// Check to ensure a term with the same name doesn't exist,
+			$search_term = $this->get_editorial_metadata_term_by( 'name', $metadata_name );
+			if ( is_object( $search_term ) && $search_term->term_id != $existing_term->term_id ) {
+				$change_error = new WP_Error( 'invalid', esc_html__( 'Name already in use. Please choose another.', 'edit-flow' ) );
+				die( esc_html( $change_error->get_error_message() ) );
+			}
 
-		if ( is_admin() && false !== stripos( get_query_var( 'orderby' ), $this->module->slug ) ) {
-			$term_slug = sanitize_key( str_replace( $this->module->slug . '-', '', get_query_var( 'orderby') ) );
-			$term = $this->get_editorial_metadata_term_by( 'slug', $term_slug );
-			$meta_key = $this->get_postmeta_key( $term );
-			set_query_var( 'meta_key', $meta_key );
-			set_query_var( 'orderby', 'meta_value_num' );
-		}
-	}
-	
-	/**
-	 * Handle the output of an editorial metadata custom column
-	 * Logic for the post types this is called on is controlled in $this->init()
-	 *
-	 * @since 0.7
-	 * @uses do_action( 'manage_posts_custom_column' ) in wp-admin/includes/class-wp-posts-list-table.php
-	 *
-	 * @param string $column_name Unique string for the column
-	 * @param int $post_id ID for the post of the row
-	 */
-	function action_manage_posts_custom_column( $column_name, $post_id ) {
-		
-		$terms = $this->get_editorial_metadata_terms();
-		// We're looking for the proper term to display its saved value
-		foreach( $terms as $term ) {
-			$key = $this->module->slug . '-' . $term->slug;
-			if ( $column_name != $key )
-				continue;
-
-			$current_metadata = $this->get_postmeta_value( $term, $post_id );
-			echo $this->generate_editorial_metadata_term_output( $term, $current_metadata );
-		}
-		
-	}
-	
-	/**
-	 * If the Edit Flow Calendar is enabled, add viewable Editorial Metadata terms
-	 *
-	 * @since 0.7
-	 * @uses apply_filters( 'ef_calendar_item_information_fields' )
-	 *
-	 * @param array $calendar_fields Additional data fields to include on the calendar
-	 * @param int $post_id Unique ID for the post data we're building
-	 * @return array $calendar_fields Calendar fields with our viewable Editorial Metadata added
-	 */
-	function filter_calendar_item_fields( $calendar_fields, $post_id ) {
+			// or that the term name doesn't map to an existing term's slug
+			$search_term = $this->get_editorial_metadata_term_by( 'slug', sanitize_title( $metadata_name ) );
+			if ( is_object( $search_term ) && $search_term->term_id != $existing_term->term_id ) {
+				$change_error = new WP_Error( 'invalid', esc_html__( 'Name conflicts with slug for another term. Please choose again.', 'edit-flow' ) );
```

Comment:
Double-escape here, can use use `__()` and let `die()` escape below

Reply:
This one's the same as custom status - it's due to the linting rules raising a concern here. Would rather not have ignores all over so I just ended up double escaping